frontend/src/components/common/info/messages/EmptySessionsTabMessage.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { type ReactElement } from "react";
+import { Center, useStyleConfig } from "@chakra-ui/react";
+
+import DistributeAssessmentsIllustration from "../../../../assets/illustrations/distribute-assessments.svg";
+import type { TestSessionStatus } from "../../../../types/TestSessionTypes";
+import MessageContainer from "../MessageContainer";
+
+const BODY_PARAGRAPHS_BY_STATUS: Record<TestSessionStatus, string[]> = {
+  UPCOMING: ["Create an assessment to view it here."],
```

Comment:
newly created assessment doesn't necessarily go to upcoming (could be active right away)